### ISSUE TITLE
Options to for GFF database construction and use could improve Liftoff in some cases.

### ISSUE THREAD

**Issue by JohnUrban** [agshumate/Liftoff#115]
Hello,

Thanks for the wonderful, absolutely fantastic program.

I've used it for a regular liftover job from one genome version to another for a non-model organism.. and it was just... so good. Thanks!

I'm now using it again for a different non-model species for which there is a previous genome and annotation. This time, I'm somewhat using it as a metric to evaluate multiple de novo assemblies. As a quick estimation of quality and completeness of each de novo assembly or at different stages of the assembly process (e.g. before and after polishing), I just look at how many genes don't map -- fewer missing genes is better as a rule of thumb, like a much faster BUSCO. Then I get the added benefit of the lifted-over annotation to further inspect.

When doing this on a HPC compute cluster, I can run liftoff on many de novo assemblies at the same time. However, a problem arises. Since I am pointing them to the same GFF, Liftoff builds the GFF database in the same location with the same name. This creates conflict and overwriting, etc. It usually expresses itself as the following error:
```local variable 'feature_db' referenced before assignment ```

The quick and simple solution for me at the moment is to just copy the GFF to the local working directory for each job, and run liftoff on that unique copy ((this is what I'm trying now, and I am assuming it will be a fine solution)). However, it occurred to me that two options in LiftOff could anticipate this use case, and prevent the error:
- One would be very simple -- just an option telling LiftOff where to build the database -- e.g. I'd just direct it to the local working directory for each job.
- A second option would be to enable a "build database only" feature for liftoff that could be run once before running the many parallel jobs that would just all use the same pre-built database.

A final thought on this is that when running dozens of jobs in parallel, all are almost certainly affected by this in some way, but only a minority fail and raise an error. It might be desirable to have liftoff raise an error when there is already a database present with the same name at the same location it is about to try to create -- i.e. stop the run and complain, or at least warn that this is happening.

Anyway that is all. Those would be cool options in my opinion, but I understand if its not a priority -- especially since I already named a suitable workaround. And apologies for the monologue if this is just a case of me not RTFM.

Best,

John
